src/InboundServer/index.js

```diff
@@ -173,52 +159,17 @@ class InboundServer extends EventEmitter {
         await this._validator.initialise(apiSpecs);
         await this._api.start();
         await new Promise((resolve) => this._server.listen(this._conf.inbound.port, resolve));
-        this._logger.log(`Serving outbound API on port ${this._conf.inbound.port}`);
+        this._logger.log(`Serving inbound API on port ${this._conf.inbound.port}`);
     }
 
     async stop() {
-        if (this._server) {
+        if (this._server.listening) {
             await new Promise(resolve => this._server.close(resolve));
-            this._server = null;
-        }
-        if (this._api) {
-            await this._api.stop();
-            this._api = null;
         }
+        await this._api.stop();
         this._logger.log('inbound shut down complete');
     }
 
-    async reconfigure(conf, logger, cache) {
```

Comment:
Glad this is gone. I found this whole reconfigure logic confusing better to just  stop and start with a new config.